gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -464,9 +464,6 @@ MM_ParallelGlobalGC::masterThreadGarbageCollect(MM_EnvironmentBase *env, MM_Allo
 	
 	sweep(env, allocDescription, rebuildMarkBits);
 
-	if (_extensions->processLargeAllocateStats) {
```

Comment:
I believe when you updated to the latest you lost this recent change. Or were you explicitly removing it?

Reply:
NVM I see that it just moved down below